app/scripts/utils/querystring.es6

```diff
@@ -0,0 +1,47 @@
+import { Ok, Err } from 'results';
+import { Result } from './functional';
+import warn from './warn';
+
+
+/**
+ * Get a CKAN-friendly query string
+ * @param {any} thing The right-hand side of a k=v in the querystring
+ * @returns {string} A url-ready right-hand side for the querystring
+ */
+function queryVal(thing, recursed = false) {
+  if (typeof thing === 'string') {
+    if (/,/.test(thing)) {
```

Comment:
probably! I wonder if lodash has a `contains` method or something, I hate writing indexOf for contains...